### ISSUE TITLE
A11y_Deploymentconfigurationtemplate_ScreenReader:Table mark up is missing under AKS-dev-test.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7933]
**GitHubTags**: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.2.3;#WCAG4.1.2;#WCAG4.2.1#DesktopApp;#Keyboard;#VoiceOver;#Mac;
**Environment Details:**Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6.check the I accept check box and click on select button.
7. Deployment template configuration screen gets open.
8. Start VoiceOver and navigate with the help of ctrl + option 
+ forward arrow up and down arrow and observe the announcement.
Actual:
Issue 1. Table mark up is missing for the "Aks -dev-test tab" and the remaining tab on the screen.
Issue 2. Table name should be announced by the NVDA.
Issue 3. Row and column reference should be announced by the NVDA.
Issue 4. column header should be marked as a column header .
Expected:
Issue 1.There should be table mark up "Aks -dev-test tab" and the reaming tab on the screen.
Issue 2. Table name should be announced by the NVDA.
Issue 3. Row and column reference should be announced by the NVDA.
Issue 4. column header should be marked as a column header .
User Impact:
In Aks -dev-test tab below content should be marked as table as user will unable to understand the relation with the correspondent data.
Screen reader User will
Attachment for Reference:
[A11y_Deployment configuration template_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3760489/A11y_Deployment.configuration.template_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes
